alacritty.yml

```diff
@@ -480,62 +481,118 @@
 # - `action`: Execute a predefined action
 #
 #   - ToggleViMode
+#       Enable or Disable Using Vi key bindings
 #   - SearchForward
+#       Start a forward buffer search
 #   - SearchBackward
+#       Start a backward buffer search
 #   - Copy
+#       Copy the selection to the clipboard
 #   - Paste
+#       Paste from the system clipboard
 #   - PasteSelection
+#       Paste from the selection buffer
 #   - IncreaseFontSize
+#       Increase font size
 #   - DecreaseFontSize
+#       Decrease font size
 #   - ResetFontSize
+#       Reset font size to config value
 #   - ScrollPageUp
+#       Scroll Exactly One Page Up
 #   - ScrollPageDown
+#       Scroll Exactly One Page Down
 #   - ScrollHalfPageUp
+#       Scroll a half page up
 #   - ScrollHalfPageDown
+#       Scroll a half page down
 #   - ScrollLineUp
+#       Scroll one line up
 #   - ScrollLineDown
+#       Scroll one line down
 #   - ScrollToTop
+#       Scroll all the way to the top
 #   - ScrollToBottom
+#       Scroll all the way to the bottom
 #   - ClearHistory
+#       Clear the display buffer(s) to remove history
```

Comment:
Are you aware of what this does? Otherwise documenting it might be a bit difficult. The statement "clear the display buffer(s)" seems a bit strange to me.

There's two buffers we have in Alacritty, the primary and alternate screen buffer. The alternate screen buffer never has any history so it's irrelevant here. So talking about buffer***s*** makes no sense in this case. So all this does is clear the **history** of the primary screen buffer. This has no effect on the visible content.

I'd propose something like this:

```suggestion
#       Remove the terminal's scrollback history
```